.github/CODEOWNERS

```diff
@@ -25,7 +25,7 @@
 
 # Hedera Node Root Protections
 /hedera-node/                                   @hashgraph/hedera-services
-/hedera-node/README.md                          @hashgraph/hedera-services @hashgraph/devops-ci @hashgraph/release-engineering-managers
+/hedera-node/README.md                          @hashgraph/hedera-services
```

Comment:
Is this line still needed? `@hashgraph/hedera-services` is responsible for the folder.